app/src/main/java/org/fossasia/susi/ai/activities/MainActivity.java

```diff
@@ -1078,7 +1078,7 @@ public void onFailure(Call<SusiResponse> call, Throwable t) {
 
     private void updateDatabase(final long prevId, final String message, final boolean isDate, final String date, final String timeStamp , final boolean mine,
```

Comment:
No, I meant that you should write the documentation for methods in the Java class itself. 
Something like this, 

```
/**
 * Validates a chess move.
 *
 * Use {@link #doMove(int theFromFile, int theFromRank, int theToFile, int theToRank)} to move a piece.
 *
 * @param theFromFile file from which a piece is being moved
 * @param theFromRank rank from which a piece is being moved
 * @param theToFile   file to which a piece is being moved
 * @param theToRank   rank to which a piece is being moved
 * @return            true if the move is valid, otherwise false
 */
boolean isValidMove(int theFromFile, int theFromRank, int theToFile, int theToRank) {
    // ...body
}
```